src/plays/2048/Game2048Components/Modal.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { FaRegQuestionCircle } from 'react-icons/fa';
+import "../Game2048Styles/Modal.css";
+
+export default function Modal() {
+  const [modal, setModal] = useState(false);
+
+  const toggleModal = () => {
+    setModal(!modal);
+  };
+
+  if (modal) {
+    document.body.classList.add('active-modal')
+  } else {
+    document.body.classList.remove('active-modal')
+  }
```

Comment:
@Shivam-Katare , Can we add this logic in a useEffect and add the `modal` as a dependency, so whenever the modal state changes this logic gets executed?